### PR TITLE
release 2.1 modifications

### LINKED ISSUE #968: Release 2.1
This is to prepare the release 2.1, which is announced in a paper. @damienpicard can you check open pull requests before we proceed?

### PR DESCRIPTION
**Pull request by Mathadon**
closes #968 

this should not be merged yet

### CODE REVIEW

**Comment by Mathadon** [comment]
@damienpicard can you already review this? But I'll merge it later.

**Comment by damienpicard** [comment]
@Mathadon changes are fine for me.

**Comment by damienpicard** [comment]
Only scheduling the release by tomorrow seems optimistic to me?

**Comment by Mathadon** [comment]
yes, that was a provisional date. Can still change it!

File: IDEAS/package.mo [hunk 1]
```diff
@@ -9,9 +9,9 @@ import SI = Modelica.SIunits;
 annotation (
   uses(Modelica(version="3.2.2")),
   Icon(graphics),
-  version="2.0.0",
-  versionDate="2018-09-27",
-  dateModified = "2018-09-27",
+  version="2.1.0",
+  versionDate="2019-02-27",
```
Comment (Mathadon) [inline IDEAS/package.mo]:
```suggestion
  versionDate="2019-02-28",
```

File: IDEAS/package.mo [hunk 2]
```diff
@@ -9,9 +9,9 @@ import SI = Modelica.SIunits;
...
+  dateModified = "2019-02-27",
```
Comment (Mathadon) [inline IDEAS/package.mo]:
```suggestion
  dateModified = "2019-02-28",
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
-IDEAS v2.0
+IDEAS v2.1
 ============
 [![Build Status](https://travis-ci.org/open-ideas/IDEAS.svg?branch=master)](https://travis-ci.org/open-ideas/IDEAS)
 
 Modelica model environment for Integrated District Energy Assessment Simulations (IDEAS), allowing simultaneous transient simulation of thermal and electrical systems at both building and feeder level.
 
 ## Release history
++ February 27th, 2019: IDEAS v2.1 has been released.
```
Comment (Mathadon) [inline README.md]:
```suggestion
+ February 28th, 2019: IDEAS v2.1 has been released.
```

File: ReleaseNotes.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
+## IDEAS v2.1
+IDEAS 2.1 has been released on February 26th, 2019. The most important changes are:
```
Comment (Mathadon) [inline ReleaseNotes.md]:
```suggestion
IDEAS 2.1 has been released on February 28th, 2019. The most important changes are:
```

**Pull request merged**